src/main/resources/assets/js/components/Marathon.jsx

```diff
@@ -456,13 +521,13 @@ define([
           <div className="container-fluid">
             <TogglableTabsComponent activeTabId={this.state.activeTabId} >
               <TabPaneComponent id="apps">
-                <button type="button" className="btn btn-success navbar-btn"
-                    onClick={this.showNewAppModal} >
-                  + New App
-                </button>
+                <a href="#newapp">
```

Comment:
Much better. BTW it's allowed in HTML5 to wrap block elements with the a element now.